lib/commands/element.js

```diff
@@ -249,7 +249,23 @@ const commands = {
 
     const atomsElement = this.getAtomsElement(el);
     await this.executeAtom('click', [atomsElement]);
+
+    if (this.opts.sendKeyStrategy !== 'oneByOne') {
+      await this.setValueWithWebAtom(atomsElement, value);
+      return;
+    }
+    for (const v of prepareInputValue(value)) {
+      await this.setValueWithWebAtom(atomsElement, v);
+    }
+  },
+
+  /**
+   * Set value with Atom for Web.
+   * @this {XCUITestDriver}
```

Comment:
would it be possible to also add type definitions for args?

Reply:
Yeah, I missed adding them...